extensions/MESA/EGL_MESA_drm_image.txt

```diff
@@ -66,6 +66,7 @@ New Tokens
 
         EGL_DRM_BUFFER_USE_SCANOUT_MESA		0x0001
         EGL_DRM_BUFFER_USE_SHARE_MESA		0x0002
+        EGL_DRM_BUFFER_USE_CURSOR_MESA	0x0004
```

Comment:
We normally name bitfield enums 'EGL_name_BIT[_vendor]', but that ship has probably sailed with this extension. However, the new bit does need to be added to egl.xml as part of this PR.

Reply:
You're right, I completely forgot that.
I should let you regenerate the headers though, right?

Reply:
Yes, I'll regenerate them when this is accepted. Thanks!